SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1028,6 +1028,39 @@ void USpatialNetDriver::ProcessRemoteFunction(
 		return;
 	}
 
+	// Copied from UNetDriver::ProcessRemoteFunctionForChannel to copy pass-by-ref
+	// parameters from OutParms into Parameters's memory.
+	if (Stack == nullptr)
+	{
+		// Look for CPF_OutParm's, we'll need to copy these into the local parameter memory manually
+		// The receiving side will pull these back out when needed
+		for (TFieldIterator<UProperty> It(Function); It && (It->PropertyFlags & (CPF_Parm | CPF_ReturnParm)) == CPF_Parm; ++It)
+		{
+			if (It->HasAnyPropertyFlags(CPF_OutParm))
+			{
+				if (OutParms == NULL)
```

Comment:
```suggestion
				if (OutParms == nullptr)
```
Yes, I know it's copied, but still.

Reply:
Good find.